OpenRA.Mods.Common/Traits/World/StartGameNotification.cs

```diff
@@ -30,7 +31,7 @@ public StartGameNotification(StartGameNotificationInfo info)
 
 		public void WorldLoaded(World world, WorldRenderer wr)
 		{
-			Sound.PlayNotification(world.Map.Rules, null, "Speech", info.Notification, null);
+			Sound.PlayNotification(world.Map.Rules, null, "Speech", info.Notification, (world.RenderPlayer == null ? world.Players.First() : world.RenderPlayer).Country.Race);
```

Comment:
Just keep this simple

```csharp
world.RenderPlayer != null ? world.RenderPlayer.Country.Race : null
```

Why the extra hack for spectators who did not select any player vision? They are just watching so "Battlecontrol initialized" is misleading anyway.